packages/lib/src/vis/shared/AxisSystem.tsx

```diff
@@ -26,35 +27,38 @@ function AxisSystem(props: Props) {
   );
 
   return (
-    <Overlay
-      className={styles.axisSystem}
-      style={{
-        // Take over space reserved for axis by VisCanvas
-        top: -axisOffsets.top,
-        left: -axisOffsets.left,
-        width: width + axisOffsets.left + axisOffsets.right,
-        height: height + axisOffsets.bottom + axisOffsets.top,
-        gridTemplateColumns: `${axisOffsets.left}px 1fr ${axisOffsets.right}px`,
-        gridTemplateRows: `${axisOffsets.top}px 1fr ${axisOffsets.bottom}px`,
-      }}
-    >
-      {title && <p className={styles.title}>{title}</p>}
-      <Axis
-        type="abscissa"
-        config={abscissaConfig}
-        domain={xVisibleDomain}
-        canvasSize={canvasSize}
-        svgSize={{ width, height: axisOffsets.bottom }}
-      />
-      <Axis
-        type="ordinate"
-        config={ordinateConfig}
-        domain={yVisibleDomain}
-        canvasSize={canvasSize}
-        svgSize={{ width: axisOffsets.left, height }}
-        flipAxis
-      />
-    </Overlay>
+    // Append to `canvasWrapper` instead of default container `r3fRoot`, which hides overflow
+    <Html container={gl.domElement.parentElement?.parentElement || undefined}>
```

Comment:
This is the key change. We go up one more element in the DOM to bypass the hidden overflow.